services/order.py

```diff
@@ -0,0 +1,32 @@
+from django.db import transaction
+
+from db.models import Ticket, Order, User
+
+
+def create_order(
+        tickets: list[{Ticket}],
+        username: str,
+        date: str = None
+) -> None:
+    with transaction.atomic():
+        order = Order.objects.create(user=User.objects.get(username=username))
```

Comment:
Use get_user_model() instead of User. Check out the [checklist](https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md#code-efficiency).